settings.py

```diff
@@ -19,6 +19,11 @@
     'debug_toolbar',
 )
 
+# Override logging config to enable DEBUG logs for (almost) everything.
+LOGGING['root'] = {'handlers': ['mozlog'], 'level': logging.DEBUG}
```

Comment:
this is the most explicit, but I suppose `LOGGING['root']['level'] = logging.DEBUG` works too